vspace/src/main/java/edu/asu/diging/vspace/core/services/IExhibitionAboutPageManager.java

```diff
@@ -0,0 +1,24 @@
+package edu.asu.diging.vspace.core.services;
+
+import java.util.List;
+
+import edu.asu.diging.vspace.core.model.impl.ExhibitionAboutPage;
+/**
+ * IExhibitionAboutPageManager allows to store and manage {@link ExhibitionAboutPage}.
+ * @author Avirup Biswas
+ *
+ */
+public interface IExhibitionAboutPageManager {
+    
+    /**
+     * This method fetches all {@link ExhibitionAboutPage} entries and returns them
+     */
+    List<ExhibitionAboutPage> findAll();
+    
+    /**
+     * This method stores Exhibition About page information and returns the {@link ExhibitionAboutPage} which is being stored
+     * @param {@link ExhibitionAboutPage} This object contain the value of Exhibition About page information
```

Comment:
This is grammatically incorrect and a little confusing. 
- should be "contains"
- do you mean the Exhibition about page values or Exhibition about page information?
- add "to be stored" to make clear what information is referred to